actions/build_deploy_pex/action.yml

```diff
@@ -16,13 +16,31 @@ inputs:
     description: 'Directory for built artifacts'
     required: false
     default: '/tmp/build'
-
+  force_rebuild_deps:
+    description: 'Whether to rebuild the dependencies, even if requirements.txt and setup.py did not change'
+    required: false
+    default: 'false'
 
 runs:
   using: "composite"
 
   steps:
-    - run: echo "ACTION_REPO=$GITHUB_ACTION_PATH/../../" >> $GITHUB_ENV
+    - name: Set ACTION_REPO
+      run: >
+        echo "ACTION_REPO=$GITHUB_ACTION_PATH/../../" >> $GITHUB_ENV
+      shell: bash
+
+    - name: Set deps-cache-tag-read
+      # Don't use cached deps if instructed
+      if: ${{ inputs.force_rebuild_deps != 'true' }}
+      # For PR commits, use the target branch name as the cache-tag, for other commits, use the branch name itself
+      run: echo "FLAG_DEPS_CACHE_TAG_READ=--deps-cache-tag-read=${{ github.repository }}/${{ github.base_ref && github.base_ref || github.ref_name }}" >> $GITHUB_ENV
```

Comment:
Is "github.base_ref && github.base_ref || github.ref_name " the equivalent of "a if a else b"? Maybe some parens would help clarify if that's an option?